lib/gears/screen.lua

```diff
@@ -0,0 +1,29 @@
+---------------------------------------------------------------------------
+-- @author Uli Schlachter
+-- @copyright 2016 Uli Schlachter
+-- @release @AWESOME_VERSION@
+-- @classmod gears.screen
+---------------------------------------------------------------------------
+
+local screen = screen
+
+local module = {}
+
+--- Call a function for each existing and created-in-the-future screen.
+-- @tparam function func The function to call.
+function module.connect_for_each_screen(func)
```

Comment:
I am fine with this, but I wonder if `added` couldn't simply be sent for every screens when Awesome starts. It would avoid this helper function.